ansible/eda-server
# [AAP-9977] Implement project sync

## Testing instructions:

1. Create Project:

    ```
    http -a admin:admin http://localhost:8000/api/eda/v1/projects/ \
        name=Sample url=https://github.com/<username>/event-driven-ansible.git
    ```

2. Create a Decision Environment and an Activation.

3. Update project and push changes to the repository.

4. Sync project:

    ```
    http -a admin:admin POST http://localhost:8000/api/eda/v1/projects/1/sync/
    ```

5. Verify that:
    * Rulebooks are updated
    * Rules are updated
    * Rulesets in Activations are udpated
    * Ruleset id in Activations referencing deleted rulebooks are set to NULL.
